pool/web/view/page/page_admin_custom_fields.ml

```diff
@@ -597,36 +596,74 @@ let detail
 
 let index field_list group_list current_model Pool_context.{ language; csrf; _ }
   =
+  let grouped, ungrouped = Custom_field.group_fields group_list field_list in
   let thead = Message.Field.[ Some Title; Some CustomFieldGroup; None ] in
+  let field_name field =
+    let open Custom_field in
+    field
+    |> name
+    |> Name.find_opt language
```

Comment:
Idea: How about adding a `find_opt_or` function to models with a `find_opt` possibility?